apps/src/code-studio/components/ModelManagerDialog.jsx

```diff
@@ -49,6 +51,9 @@ export default class ModelManagerDialog extends React.Component {
       url: '/api/v1/ml_models/names',
       method: 'GET'
     }).then(models => {
+      if (this.props.levelbuilderModel && this.props.levelbuilderModel.id) {
```

Comment:
this is a great place to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)! (charlie just enabled this in #39543, so we couldn't use that feature until now. you'll likely need to pull/merge staging into your branch as well)

Reply:
Well, isn't that handy? Thanks Maddie (and Charlie!). Updating...